.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- I think this project requires us to ignore test.md file as well, just the way you added node modules.

 As suggested by the previous reviewer, kindly add `test.md` to your ` .gitignore` file, that way, when you push to hithub, test.md file wouldn't push too. 🌸

Reply:
Hi, I have added test.md to my .gitignore file. Do I need to re-submit to get an approval? Thanks a lot 🙏

Reply:
If you are 100% sure that it's been added, you can go ahead and submit for review. But please check properly, because i am not sure you have added it. 😄

Reply:
How about now? Could you please check it again? Because this is my last chance to get approval. Thanks 😊🙏

Reply:
Sure, you have done well 👍🏽. You can submit